src/quo/components/tabs/tabs/view.cljs

```diff
@@ -158,15 +158,12 @@
              :on-change
              :scroll-on-press?
              :size)
-            (when scroll-on-press?
-              {:initial-scroll-index (utils.collection/first-index #(= @active-tab-id (:id %)) data)})
```

Comment:
While using `initial-scroll-index` without adding a `get-item-layout` function, it results in buggy behavior. In this case, implementing `get-item-layout` is not possible because the item width can't be identified beforehand, as the width depends on the tab title.